sanic_jinja2/__init__.py

```diff
@@ -64,32 +77,41 @@ def update_request_context(self, request, context):
         context.setdefault('get_flashed_messages',
                            partial(self._get_flashed_messages, request))
 
+
     async def render_string_async(self, template, request, **context):
         self.update_request_context(request, context)
         return await self.env.get_template(template).render_async(**context)
 
+
     async def render_async(self, template, request, **context):
         return html(await self.render_string_async(template, request,
                                                    **context))
 
+    arender = render_async
+
+
     def render_source(self, source, request, **context):
         self.update_request_context(request, context)
         return self.env.from_string(source).render(**context)
 
+
     def render_string(self, template, request, **context):
         self.update_request_context(request, context)
         return self.env.get_template(template).render(**context)
 
+
     def render(self, template, request, **context):
         return html(self.render_string(template, request, **context))
 
+
```

Comment:
why put two newlines in class methods?

Reply:
It was settings for my editor. I'll fix it.